packages/@lwc/integration-karma/test/light-dom/slotting/x/forwardedSlot/forwardedSlot.html

```diff
@@ -0,0 +1,7 @@
+<template lwc:render-mode="light">
+    <x-light-container>
+        <slot slot="upper" name="upper"></slot>
+        <slot>Default slot not yet forwarded</slot>
+        <slot slot="lower" name="lower"></slot>
```

Comment:
Can we add a test for default slot content here as well? I.e. the case where `<x-forwarded-slot>` is not passed any slots at all.